frontend/public/components/_storage-class-form.scss

```diff
@@ -1,3 +1,4 @@
 .create-storage-class-form__checkbox {
   width: inherit;
+  height: 100%;
```

Comment:
this is a core css, please find another way, we should not touch the core component without a ** very ** good reason

Reply:
you right, i didnt notice its in /public, fixed